js/childrenPopUp.js

```diff
@@ -0,0 +1,75 @@
+var Toast = function(config) {
+    for(var key in config) {
+        this[key] = config[key];
+    }
+
+    this.__proto__ = PopUpNotifications;
```

Comment:
You shouldn't be using __proto__ word for Inheritance. Please, use prototype instead